src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -164,23 +163,25 @@ public static String sanitizeForHtml(String unsanitizedString) {
     /**
      * Sanitizes a list of strings for inserting into HTML.
      */
-    public static List<String> sanitizeForHtml(List<String> list) {
-        List<String> sanitizedList = new ArrayList<>();
-        for (String str : list) {
-            sanitizedList.add(sanitizeForHtml(str));
+    public static List<String> sanitizeForHtml(List<String> unsanitizedStringList) {
+        if (unsanitizedStringList == null) {
+            return null;
         }
-        return sanitizedList;
+        return unsanitizedStringList.stream()
+                .map(s -> sanitizeForHtml(s))
+                .collect(Collectors.toCollection(ArrayList::new));
     }
 
     /**
      * Sanitizes a set of strings for inserting into HTML.
      */
-    public static Set<String> sanitizeForHtml(Set<String> set) {
-        Set<String> sanitizedSet = new TreeSet<>();
-        for (String str : set) {
-            sanitizedSet.add(sanitizeForHtml(str));
+    public static Set<String> sanitizeForHtml(Set<String> unsanitizedStringSet) {
+        if (unsanitizedStringSet == null) {
+            return null;
         }
-        return sanitizedSet;
+        return unsanitizedStringSet.stream()
+                .map(s -> sanitizeForHtml(s))
+                .collect(Collectors.toCollection(HashSet::new));
```

Comment:
What happens if you use `TreeSet` like in the original?

Reply:
It doesn't really matter since the expected return is of type `Set<String>` and both `HashSet` and `TreeSet` implement `Set`. I just changed it to `HashSet` to avoid an unnecessary import. I did not see any point to using a `TreeSet` since we don't require any ordering guarantee (see [this thread](https://stackoverflow.com/questions/1463284/hashset-vs-treeset)). If you see the original for `desanitizeFromHtml` below, they use `HashSet`!

Do you want me to change it back to `TreeSet`?

Reply:
That's true; it's fine then. Actually I just realised this method isn't even used at all.

Reply:
If the method is not used, why not get rid of it?

Reply:
I wasn't sure about best practices regarding this because this method already existed before I worked on this PR and I wasn't sure if I have the prerogative to be removing someone else's utility method. Shall I get rid of it?

Reply:
@darrenwee you don't need to be hesitant in removing unused code.